go-controller/pkg/util/multi_network_test.go

```diff
@@ -1048,11 +1166,14 @@ func TestGetPodNADToNetworkMappingWithActiveNetwork(t *testing.T) {
 				primaryUDNNetInfo,
 			)
 
-			if err != nil {
+			if test.expectedError != nil {
+				g.Expect(err).To(gomega.HaveOccurred(), "unexpected success operation, epecting error")
 				g.Expect(err).To(gomega.MatchError(test.expectedError))
```

Comment:
aren't these redundant ? i.e. I'd just leave the expectation with the `MatchError` matcher.